pkg/controller/controller_test.go

```diff
@@ -1014,9 +1014,9 @@ func TestFSTypeProvision(t *testing.T) {
 		},
 	}
 
-	for k, tc := range testcases {
+	for k := range testcases {
 		t.Run(k, func(t *testing.T) {
-			runFSTypeProvisionTest(t, k, tc, requestedBytes, driverName, "" /* no migration */)
+			runFSTypeProvisionTest(t)
```

Comment:
for every `k, testcase := range testcase` we're trying to test `testcase` however I don't see any reference to it here